internal/common/consts.go

```diff
@@ -57,6 +57,10 @@ const (
 	InstallConfigCM = "cluster-config-v1"
 	// InstallConfigCMNamespace cm namespace
 	InstallConfigCMNamespace = "kube-system"
+
+	// Bump this every time the seed format changes in a backwards incompatible way
+	SeedFormatVersion = 1
```

Comment:
Spacing change for go fmt:
```suggestion
	SeedFormatVersion  = 1
```